### ISSUE TITLE
Embed Iframe into Streamsync

### ISSUE THREAD

**Issue by duckunz** [streamsync-cloud/streamsync#308]
Dear all,

I want to embed a web page into streamsync using html element but it wont work

For example I want the page http://24h.com.vn

Please help

**Comment by FabienArcellier** [comment]
You can also use the existing html component with the following parameters. It should work

![image](https://github.com/streamsync-cloud/streamsync/assets/159559/0883aecc-8d9d-49f2-adbb-9bb6c710de02)

**Comment by duckunz** [comment]
Thanks a lot for your help. It works for normal website but for me I want to ask more specific case.

For example if I have a localhost (like http://localhost:8888/tree) website or LAN network url (like http://10.21.210.47:8888/tree) - jupyter notebook for specific. 

When I inserted {"src":"http://10.21.210.47:8888/tree"} into streamsync, I got 10.21.210.47 refused to connect.

Could you help me to fix this.

Many thanks!

**Comment by raaymax** [comment]
I don't understand what you are trying to do but if you are using the browser from inside of the local network you should be able to access those pages. We need more information about your use-case.
Join our discord server, I'm sure we will be able to find solution for you there ;)
https://discord.gg/nMn9YPRGSY

**Comment by FabienArcellier** [comment]
Jupyter has several security mechanisism active by default to improve daily security.

If you need to embed a jupyter notebook into streamsync, you will find all the response in this page : https://jupyter-notebook.readthedocs.io/en/6.2.0/public_server.html#running-a-public-notebook-server and in section https://jupyter-notebook.readthedocs.io/en/6.2.0/public_server.html#embedding-the-notebook-in-another-website

**Comment by FabienArcellier** [comment]
For this example, I bypass many security rule, generating a configuration file to disable the `Content-Security-Policy`, then running the command.

**don't use this configuration in unsafe network, I give you info to help you find the best way to configure your policy and showing it's working**

```bash
jupyter notebook --ServerApp.disable_check_xsrf=true --ServerApp.allow_origin='*' --ServerApp.token=''
```

![image](https://github.com/streamsync-cloud/streamsync/assets/159559/29c663e9-c572-45be-8441-caa81a00aa7a)

### Generate jupyter configuration

```bash
jupyter notebook --generate-config
```

*/home/${USER}/.jupyter/jupyter_notebook_config.py*
```python
c.NotebookApp.tornado_settings = {
    'headers': {
        'Content-Security-Policy': ""
    }
}
```

**Comment by ramedina86** [comment]
The _iFrame_ component was released in 0.4.0.
